lib/process-services-cloud/src/lib/form/components/attach-file-cloud-widget/attach-file-cloud-widget.component.ts

```diff
@@ -127,30 +118,20 @@ export class AttachFileCloudWidgetComponent extends UploadCloudWidgetComponent
     }
 
     downloadContent(file: Node): void {
-        this.processCloudContentService
-            .getRawContentNode(file.id, this.field.form.contentHost)
-            .subscribe(
-                (blob: Blob) => {
-                    this.processCloudContentService.downloadNodeContent(
-                        blob,
-                        file.name
-                    );
-                },
-                () => {
-                    this.logger.error(
-                        'Impossible retrieve content for download'
-                    );
-                }
-            );
+        this.processCloudContentService.downloadFile(
```

Comment:
what is the file name of the downloaded file? Before we were passing `file.name`

Reply:
In fact it is not working, the server defines the name and client cannot change it (content-disposition header)